variants/aws-ecs-2/Cargo.toml

```diff
@@ -12,6 +12,7 @@ grub-set-private-var = true
 unified-cgroup-hierarchy = true
 uefi-secure-boot = true
 xfs-data-partition = true
+systemd-networkd = true
```

Comment:
Need to also add this to `aws-ecs-2-nvidia`. Wouldn't hurt to add it to `*-dev` in a separate commit.